apps/laboratory/tests/multichain/multichain-ethers-solana-email-siwe.spec.ts

```diff
@@ -59,7 +59,7 @@ test('it should switch to different namespace', async () => {
   const chainName = 'Solana'
 
   await page.switchNetwork(chainName)
-  await page.openNetworks()
+  await validator.expectUnauthenticated()
```

Comment:
In this case the expected behavior of after network switch is to keep the switch network view opened, right? I've changed to `expectUnauthenticated` because when switching to non evm with siwe, the current behavior is no validation of signature so unauthenticated.